cmd/query/app/static_handler.go

```diff
@@ -112,75 +116,92 @@ func NewStaticAssetsHandler(staticAssetsRoot string, options StaticAssetsHandler
 	return h, nil
 }
 
-func loadAndEnrichIndexHTML(open func(string) (http.File, error), options StaticAssetsHandlerOptions) ([]byte, error) {
+func (sH *StaticAssetsHandler) reloadUIConfig() {
+	sH.options.Logger.Info("reloading UI config", zap.String("filename", sH.options.UIConfigPath))
+	content, err := sH.options.loadAndEnrichIndexHTML(sH.assetsFS.Open)
+	if err != nil {
+		sH.options.Logger.Error("error while reloading the UI config", zap.Error(err))
+	}
+	sH.indexHTML.Store(content)
+	sH.options.Logger.Info("reloaded UI config", zap.String("filename", sH.options.UIConfigPath))
+}
+
+func (sH *StaticAssetsHandler) loggingHandler(handler http.Handler) http.Handler {
```

Comment:
please keep the methods where they were. This reordering is introducing changes to the PR that are unnecessary and make it much more difficult to review.